scipy/stats/tests/test_stats.py

```diff
@@ -5152,7 +5202,7 @@ def test_ttest_1samp_new():
     assert_almost_equal(t1[0,0],t3, decimal=14)
     assert_equal(t1.shape, (n1,n3))
 
-    t1,p1 = stats.ttest_1samp(rvn1[:,:,:], np.ones((n1,n2)),axis=2)
```

Comment:
Our convention for broadcasting (ignoring the axis identified by `axis`) begins by _prepending_ ones so that the two arrays have the same number of dimensions. Under that convention, shape `(n1, n2, n3` and `(n1, n2)` aren't broadcastable because `(n1, n2)` becomes `(1, n1, n2)`. This shouldn't have worked before.